RECIPES.md

```diff
@@ -38,44 +74,21 @@ These require:
 
 You can expand a stub, even if it's an incremental improvement. For example, you can expand a CMS guide by adding an "Integrating with Astro" section to explain how to connect the CMS to Astro. You can also provide a step-by-step section to use that integration to build something. (See [Making a blog with Astro and Contentful](https://docs.astro.build/en/guides/cms/contentful/#making-a-blog-with-astro-and-contentful).)
 
 ### Add new links
 
 The CMS, Deploy, and Migrate guides all have sections for external links. You can add a new link by [creating a PR](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md).
 
-## More Recipes page
-
-This landing page shows how-to guides that don't fit in the above categories. 
-
-The Official Recipes section is for officially maintained recipes, found in the `src/content/guides/recipes` folder. Like the rest of our docs, these must meet the standards of our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and require long-term maintenance to make sure they're up to date.
-
-Community Recipes are for external links that fit the definition of a recipe: short, focused how-to guides. This lets us include community resources without requiring long-term maintenance.
-
-### Is my guide a recipe?
-
-Our recipe format is inspired by Diataxis's [how-to guides](https://diataxis.fr/how-to-guides/). Recipes:
-
-- Provide step-by-step instructions
-- Meet a user at a specific starting point (see our [Prerequisites](https://docs.astro.build/en/recipes/build-forms-api/#prerequisites) sections)
-- Explain how to accomplish a specific task
-- Use real-world examples
-
-See our [HTML Forms](https://docs.astro.build/en/recipes/build-forms-api/) recipe as an example. It teaches a specific, practical goal (how to build and handle a form). It provides an example that can be modified and incorporated into your project. It uses step-by-step instructions and keeps explanations to a minimum.
-
-### Community Recipes
-
-You can share a published recipe in our “Community Recipes” section. This includes posts on a personal or company blog, or on your preferred blogging platform like [DEV.to](https://dev.to/). This follows the IndieWeb principle of “Publish (on your) Own Site, Syndicate Elsewhere” (often abbreviated to [POSSE](https://indieweb.org/POSSE)).
-
-After you publish your article, [open a PR adding a link to this page](https://github.com/withastro/docs/edit/main/src/content/docs/en/recipes.mdx).
+## Add a new official recipe
 
-### Official Recipes
+The Official Recipes section is for officially maintained recipes, found in the `src/content/guides/recipes` folder, that do not fit into one of our pre-existing categories. Like the rest of our docs, these must meet the standards of our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and require long-term maintenance to make sure they're up to date.
```

Comment:
Another singular/plural disagreement?
```suggestion
The Official Recipes section is for officially maintained recipes, found in the `src/content/guides/recipes` folder, that does not fit into one of our pre-existing categories. Like the rest of our docs, these must meet the standards of our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and require long-term maintenance to make sure they're up to date.
```

Reply:
I think I'm actually correct here! (For a change... :sweat_smile:)

Check me:
the SECTION (sing) is for RECIPES (pl) that DO (pl) not fit....

Reply:
You're right, disregard this one and merge! 😜